content/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -67,23 +67,21 @@ After configuring your CDN, you need instruct SparkPost to encode your links usi
 ---
 ## Step by Step Guide with CloudFlare
 
-_Updated December 2021. Uses a simpler forwarding method without the need for custom page rules. Images and descriptions follow the current CloudFlare web UI._
+_Updated June 2023. Uses a simpler forwarding method without the need for custom page rules. Images and descriptions follow the current CloudFlare web UI._
```

Comment:
Small suggestion so the text is the same as the CloudFront one. I don't think the additional information is useful.
```suggestion
_Updated for Cloudflare web UI as of June 2023._
```

Reply:
Done